src/main/resources/openapi/sqs/print-api-sqs-messaging.yaml

```diff
@@ -63,62 +63,62 @@ components:
         issueDate:
```

Comment:
`issueDate` - a question for @kristerbone ....
Is VCA (and the other future apps) responsible for generating the Issue Date? Or, in the same way that `print-api` is responsible for generating the VAC number and suggested expiry date (issue date + 10 years), should `print-api` be responsible for generating the Issue Date ?

It seems to me that perhaps the date the certificate is issued is the date it was sent to print at the print provider, and not necessarily the date at which the source *application* [sic] sent it to the `print-api`

I'm probably splitting hairs over nothing here, but thoughts @kristerbone ?

Reply:
Happy for print-api to generate this. As above, if there does become another source, would keep issue date consistent.

Reply:
👍 Cool, thanks, will remove this this spec then 👍